client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./PageHeader.module.css";
+import { PropTypes } from "prop-types";
+
+const MenuDropdown = ({ onClose }) => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {
+    // TODO : API call for logout
+    navigate("/");
+  };
+
+  return (
+    <div className={styles.navDropdown}>
+      <NavButton hamburger={false} onClick={onClose} />
+      <div className={styles.menuItemContainer}>
+        <MenuDropdownItem
+          label={"Calendar"}
+          action={() => navigate("/calendar")}
+        />
+        <MenuDropdownItem
+          label={"Task History"}
+          action={() => navigate("/task_history")}
+        />
+        <MenuDropdownItem
+          label={"Insights"}
+          action={() => navigate("/insights")}
+        />
+        <MenuDropdownItem label={"Log out"} action={handleLogOut} />
+      </div>
+    </div>
+  );
+};
+
+const MenuDropdownItem = ({ label, action }) => {
+  return (
+    <button className={styles.navDropdownItem} onClick={action}>
+      {label}
+    </button>
+  );
+};
+
+const NavButton = ({ hamburger, onClick }) => {
```

Comment:
I would rename this prop to be `isClosed` instead of hamburger. Hamburger is a bit too vague and doesn't mean a lot compared to something more descriptive such as `isClosed`

Reply:
Renamed.